invenio_files_rest/views.py

```diff
@@ -814,6 +814,10 @@ def get(self, bucket=None, key=None, version_id=None, upload_id=None,
             return self.multipart_listparts(bucket, key, upload_id)
         else:
             obj = self.get_object(bucket, key, version_id)
+            if current_app.config['FILES_REST_OFFLOAD_STREAMING_EXTERNALLY']:
+                response_constructor = current_app.config[
+                    'FILES_REST_STREAMING_REDIRECT_RESPONSE_FUNC']
+                return response_constructor(obj)
```

Comment:
Silly question: if we pass "only" the `obj`, and inside we do `obj.bucket`, does SQLAlchemy perform another db query? If yes, maybe we want to pass more params...